flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/nfa/NFA.java

```diff
@@ -804,7 +786,11 @@ private boolean checkFilterCondition(
 	/**
 	 * The context used when evaluating this computation state.
 	 */
-	private static class ConditionContext<T> implements IterativeCondition.Context<T> {
+	private class ConditionContext implements IterativeCondition.Context<T> {
+
+		private final CepTimerService timerService;
+
+		private final long eventTimestamp;
```

Comment:
Rename to `eventTimeTimestamp` and have a comment with what the value is in case there is no event time timestamp.

Reply:
It is not always eventTime, it might be processing time timestamp.